docs/source/data/data_catalog.md

```diff
@@ -39,7 +39,7 @@ The following prepends are available:
 - **Azure Blob Storage / Azure Data Lake Storage Gen2**: `abfs://` - Azure Blob Storage, typically used when working on an Azure environment.
 - **HTTP(s)**: ``http://`` or ``https://`` for reading data directly from HTTP web servers.
 
-`fsspec` also provides other file systems, such as SSH, FTP and WebHDFS. See the [documentation](https://filesystem-spec.readthedocs.io/en/latest/api.html#implementations) for more information.
+`fsspec` also provides other file systems, such as SSH, FTP and WebHDFS. [See the fsspec documentation for more information](https://filesystem-spec.readthedocs.io/en/latest/api.html#implementations).
```

Comment:
`fsspec` also provides other file systems, such as SSH, FTP and WebHDFS. See the [fsspec documentation] (https://filesystem-spec.readthedocs.io/en/latest/api.html#implementations) for more information.